Documentation/spelling_wordlist.txt

```diff
@@ -451,6 +452,7 @@ mainboard
 makefile
 masq
 matchPattern
+maxConnectedClusters
```

Comment:
This should not be necessary, as helm keys are automatically skipped from spell checking.

Reply:
@thorn3r Are you seeing build failures on this term?

Reply:
@thorn3r Are you seeing build failures on this term?

Reply:
@zacharysarah I was early on, but it's possible that was before I finished the helm changes. I'll reassess and remove this if its not needed

Reply:
removed this from the wordlist!